packages/playground/website/builder/builder.js

```diff
@@ -0,0 +1,925 @@
+				} else if (subProp?.type === 'object') {
+					editor
+						.getSession()
+						.insert(
+							{ row: event.end.row, column: event.end.column },
+							' {}'
+						);
+					editor.moveCursorTo(event.end.row, 2 + event.end.column);
+				}
+			}
+
+			if (
+				prevKey.length === 2 &&
+				prevKey[0] === 'step' &&
+				prevKey[1] === 'steps'
+			) {
+				editor.getSession().insert(
+					{
+						row: event.start.row,
+						column: event.start.column + 1,
+					},
+					' ""'
+				);
+				editor.moveCursorTo(event.end.row, 1 + event.end.column);
+				editor.execCommand('startAutocomplete');
+			} else if (prevKey.length === 2 && prevKey[1] === 'steps') {
+				const stepType = await getLastOfType(editor, 'step', event.end);
+				const properties = await getStepProperties(stepType);
+				const property = properties[prevKey[0]] ?? null;
+				const propType = property.type ?? null;
+				const propRef = property['$ref'];
+
+				if (propType === 'string') {
+					editor
+						.getSession()
+						.insert(
+							{ row: event.end.row, column: event.end.column },
+							' ""'
+						);
+					editor.moveCursorTo(event.end.row, 2 + event.end.column);
+				} else if (propRef === '#/definitions/FileReference') {
+					editor
+						.getSession()
+						.insert(
+							{ row: event.end.row, column: event.end.column },
+							' {}'
+						);
+					editor.moveCursorTo(event.end.row, 2 + event.end.column);
+				}
+			}
+		} else if (inserted === '[') {
+			editor
+				.getSession()
+				.insert(
+					{ row: event.end.row, column: event.start.column + 1 },
+					']'
+				);
+			return;
+		} else if (inserted === '{') {
+			editor
+				.getSession()
+				.insert(
+					{ row: event.end.row, column: event.start.column + 1 },
+					'}'
+				);
+			return;
+		} else if (inserted === ',') {
+			const nextIndent = (lines[1 + event.start.row].match(/^(\s+)/g) || [
+				'',
+			])[0];
+			if (nextIndent.length >= indent.length) {
+				return;
+			}
+			if (lines[event.start.row][-1 + event.start.column] !== '"') {
+				editor
+					.getSession()
+					.insert(
+						{ row: event.end.row, column: event.end.column },
+						'\n' + indent
+					);
+				editor.moveCursorTo(1 + event.end.row, 1 + indent.length);
+				return;
+			}
+			editor
+				.getSession()
+				.insert(
+					{ row: event.end.row, column: event.end.column },
+					'\n' + indent + '""'
+				);
+			editor.moveCursorTo(1 + event.end.row, 1 + indent.length);
+			editor.execCommand('startAutocomplete');
+		}
+	});
+
+	window.test = {
+		iframeSrc,
+		iframe,
+		textarea,
+		button,
+	};
+
+	button.addEventListener('click', () => {
+		try {
+			clearError();
+			runBlueprint(editor);
+		} catch (error) {
+			showError(error);
+		}
+	});
+
+	let prevWin;
+
+	newTab.addEventListener('click', () => {
+		runBlueprint(editor);
+		const query = new URLSearchParams();
+
+		query.set('mode', 'seamless');
+		// query.set('php', blueprint?.preferredVersions?.php);
```

Comment:
Let's remove these commented lines – they don't affect the program, but they could confuse the reader ("Why is that line commented? Is it important somehow?")

Reply:
@adamziel done: https://github.com/WordPress/wordpress-playground/pull/773/files#diff-21cb6ea89be95aa56dd4e93aada49e8ca1e7055ce64690acdf795e45fa7d44daR816